ragna/deploy/_cli/core.py

```diff
@@ -171,3 +177,215 @@ def wait_for_api() -> None:
         if process is not None:
             process.kill()
             process.communicate()
+
+
+class SourceStorageImport(BaseModel):
+    source_storage_class: ImportString
+
+
+class DocumentFilesystemHandlerImport(BaseModel):
+    document_filesystem_handler_class: ImportString
+
+
+@corpus_app.command(help=" Ingest some documents into a given corpus.")
+def ingest(
+    source_storage: Annotated[
+        str,  # typer does not support ImportString
+        typer.Option(
+            "--source-storage",
+            metavar="SOURCE_STORAGE",
+            default_factory=lambda: "ragna.source_storages.RagnaDemoSourceStorage",
+            help="Select source for storing vector embeddings.",
+        ),
+    ],
+    document_filesystem_handler: Annotated[
+        str,  # typer does not support ImportString
+        typer.Option(
+            "--document-filesystem-handler",
+            metavar="DOCUMENT_FILESYSTEM_HANDLER",
+            default_factory=lambda: "ragna.core.LocalDocument",
+            help="Select source for storing vector embeddings.",
+        ),
+    ],
+    database_url: Annotated[
+        str,
+        typer.Option(
+            "--database-url",
+            metavar="DATABASE_URL",
+            default_factory=lambda: f"sqlite:///{local_root()}/ragna.db",
+            help="URL for ragna database.",
+        ),
+    ],
+    corpus_name: Annotated[
+        Optional[str],
+        typer.Option(
+            "--corpus-name",
+            metavar="CORPUS_NAME",
+            default_factory=lambda: None,
+            help="Name of the corpus to ingest the documents into.",
+        ),
+    ],
+    documents: list[Path],
+    verbose: Annotated[bool, typer.Option("--verbose")] = False,
```

Comment:
This is only used at the moment to capture files that were not ingested, but can be extended in various ways.